### PR TITLE
Add WITF

### PR DESCRIPTION
**Pull request by Sleepdealr**
## Description

WITF Keyboard PR

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

N/A

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ramlord/witf/readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# witf
+
+![witf](imgur.com image replace me!)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Sleepdealer](https://github.com/Sleepdealr)
+* Hardware Supported: *The PCBs, controllers supported*
+* Hardware Availability: *Links to where you can find this hardware*
```
Comment (zvecr) [inline keyboards/ramlord/witf/readme.md:9]: Should be filled in

File: keyboards/ramlord/witf/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Make example for this keyboard (after setting up your build environment):
+
+    make witf:default
+
+Flashing example for this keyboard:
+
+    make witf:default:flash
```
Comment (zvecr) [inline keyboards/ramlord/witf/readme.md:17]:
```suggestion
    make ramlord/witf:default

Flashing example for this keyboard:

    make ramlord/witf:default:flash
```

File: keyboards/ramlord/witf/readme.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
+* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
+* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```
Comment (zvecr) [inline keyboards/ramlord/witf/readme.md:27]:
```suggestion
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```

File: keyboards/ramlord/witf/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,23 @@
+#include QMK_KEYBOARD_H
```
Comment (zvecr) [inline keyboards/ramlord/witf/keymaps/via/keymap.c:1]: Needs compliant licence header

```suggestion
// Copyright 2022 Sleepdealer (@Sleepdealr)
// SPDX-License-Identifier: GPL-2.0-or-later

#include QMK_KEYBOARD_H
```

File: keyboards/ramlord/witf/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,23 @@
+#include QMK_KEYBOARD_H
```
Comment (zvecr) [inline keyboards/ramlord/witf/keymaps/default/keymap.c:1]: Needs compliant licence header

```suggestion
// Copyright 2022 Sleepdealer (@Sleepdealr)
// SPDX-License-Identifier: GPL-2.0-or-later

#include QMK_KEYBOARD_H
```

File: keyboards/ramlord/witf/info.json [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["B4", "B5", "B14", "B13", "B12", "B11", "B10", "B2",  "B1", "B0", "A7", "A6"],
+        "rows": ["A1", "A2", "B15", "A8",  "A9",  "A10", "A14", "A15", "B3", "B6", "B7", "B8"]
+    },
+    "processor": "STM32F072",
```
Comment (zvecr) [inline keyboards/ramlord/witf/info.json:19]:
```suggestion
    "indicators": {
        "num_lock": "A5",
        "caps_lock": "A3",
        "scroll_lock": "A4"
    },
    "processor": "STM32F072",
```

File: keyboards/ramlord/witf/config.h [hunk 7]
```diff
@@ -0,0 +1,24 @@
+// Copyright 2022 Sleepdealer (@Sleepdealr)
```
Comment (zvecr) [inline keyboards/ramlord/witf/config.h:1]: This file should be removed

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**